tests/SpyFailure.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+A test with a spy fails due to expected call not made 
+--FILE--
+<?php
+
+require_once dirname(__DIR__) . '/xdebug_filter.php'; 
+require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
I suggest creating a `phpunit_runner.php` file that would deal with requiring the autoloader, configure the xdebug filter and define a (namespaced) `runTest` function taking the name of the fixture test to run (`SpyFailure` passed as argument).

This way, the actual logic to run PHPUnit is not duplicated in each test, making it easier to add support for PHPUnit 10.

Reply:
Done in d74643b